src/frontend/screens/Observation/ObservationView.js

```diff
@@ -1,50 +1,112 @@
 // @flow
 import React from "react";
-import { Text, Image, View, ScrollView, StyleSheet, Share } from "react-native";
+import { Text, View, ScrollView, StyleSheet, Share } from "react-native";
+import MapboxGL from "@react-native-mapbox-gl/maps";
 import ShareMedia from "react-native-share";
 import MaterialIcons from "react-native-vector-icons/MaterialIcons";
 
 import api from "../../api";
+import MapStyleProvider from "../../sharedComponents/MapStyleProvider";
 import FormattedCoords from "../../sharedComponents/FormattedCoords";
 import ThumbnailScrollView from "../../sharedComponents/ThumbnailScrollView";
-import { DetailsIcon, CategoryIcon } from "../../sharedComponents/icons";
+import { CategoryCircleIcon } from "../../sharedComponents/icons";
+import mapIcon from "../../images/mapeo-icon-transparent.png";
 import { formatDate, formatCoords } from "../../lib/utils";
 import { filterPhotosFromAttachments } from "../../context/DraftObservationContext";
-import { RED, WHITE } from "../../lib/styles";
+import {
+  BLACK,
+  RED,
+  WHITE,
+  DARK_GREY,
+  LIGHT_GREY,
+  MEDIUM_GREY
+} from "../../lib/styles";
 import { TouchableOpacity } from "../../sharedComponents/Touchables";
 import type { PresetWithFields } from "../../context/PresetsContext";
 import type { Observation } from "../../context/ObservationsContext";
 
-// const InsetMapView = ({ style }) => <View style={[style, styles.insetMap]} />;
-
 type ButtonProps = {
   onPress: () => any,
   color: string,
-  iconName: "delete-forever" | "share",
+  iconName: "delete" | "share",
   title: string
 };
 
+type MapProps = {
+  lon: number,
+  lat: number
+};
+
+const MapFeatures = ({ lat, lon }: MapProps) => {
+  const featureCollection = {
+    type: "FeatureCollection",
+    features: [
+      {
+        type: "Feature",
+        geometry: {
+          type: "Point",
+          coordinates: [lon, lat]
+        },
+        properties: {
+          id: Date.now()
+        }
+      }
+    ]
+  };
+  return (
+    <MapboxGL.ShapeSource id="observation-source" shape={featureCollection}>
+      <MapboxGL.SymbolLayer
+        id="observation-symbol"
+        style={{
+          iconImage: mapIcon,
+          iconSize: 0.5,
+          iconOffset: [0, -15]
```

Comment:
```suggestion
          iconAnchor: "bottom"
```

The icon was not aligned to the actual point. `iconAnchor: "bottom"` should fix this, _but_ it also needs the map icon to be edited because `images/mapeo-icon-transparent.png` is not cropped - the bottom point of the icon needs to be at the bottom edge of the PNG image for `iconAnchor` to work